src/main/java/seedu/address/logic/commands/CommandWord.java

```diff
@@ -21,6 +21,7 @@
     LIST("list"),
     VIEW("view"),
     SORT("sort"),
+    SHOW_REMINDER("show"),
```

Comment:
should this just be `SHOW`? none of the other commands are attached to the type of functionality they target.